06-files-input-and-output/remove_files.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import os
+
+def find_files_in_folder(extension, file_max_size, path):
+    print(
+        "Searching for files with .{extention} extension in \"{path}\""
+        " and maximum size of {size} bytes".format(
+            extention=extension,
+            path=path,
+            size=file_max_size,
+        )
+    )
+
+    file_list = []
+
+    for current_folder, subfolders, file_names in os.walk(path):
+        for file_name in file_names:
+            full_path = os.path.join(current_folder, file_name)
+            file_size = os.path.getsize(full_path)
+
+            if file_name.lower().endswith(extension) and file_size < file_max_size:
+                file_list.append(full_path)
+
+    return file_list
+
+def delete_files(fileList):
+    if file_list:
```

Comment:
I think this `if` should be outside of this function, along side the `else` you left in the main function.